incubator/binding-openapi/src/main/java/io/aklivity/zilla/runtime/binding/openapi/internal/streams/OpenapiClientFactory.java

```diff
@@ -0,0 +1,1065 @@
+
+            assert acknowledge <= sequence;
+            assert sequence >= replySeq;
+
+            replySeq = sequence + reserved;
+
+            assert replyAck <= replySeq;
+            assert replySeq <= replyAck + replyMax;
+
+            delegate.doHttpReplyData(traceId, flags, reserved, payload, extension);
+        }
+
+        private void onHttpReplyFlush(
+            FlushFW flush)
+        {
+            final long sequence = flush.sequence();
+            final long acknowledge = flush.acknowledge();
+            final long traceId = flush.traceId();
+            final int reserved = flush.reserved();
+            final OctetsFW extension = flush.extension();
+
+            assert acknowledge <= sequence;
+            assert sequence >= replySeq;
+
+            replySeq = sequence + reserved;
+
+            assert replyAck <= replySeq;
+            assert replySeq <= replyAck + replyMax;
+
+            delegate.doOpenapiReplyFlush(traceId, extension);
+        }
+
+        private void onHttpReplyEnd(
+            EndFW end)
+        {
+            final long sequence = end.sequence();
+            final long acknowledge = end.acknowledge();
+            final long traceId = end.traceId();
+            final OctetsFW extension = end.extension();
+
+            assert acknowledge <= sequence;
+            assert sequence >= replySeq;
+
+            replySeq = sequence;
+            state = OpenapiState.closingReply(state);
+
+            assert replyAck <= replySeq;
+
+            delegate.doHttpReplyEnd(traceId, extension);
+        }
+
+        private void onHttpReplyAbort(
+            AbortFW abort)
+        {
+            final long sequence = abort.sequence();
+            final long acknowledge = abort.acknowledge();
+            final long traceId = abort.traceId();
+
+            assert acknowledge <= sequence;
+            assert sequence >= replySeq;
+
+            replySeq = sequence;
+            state = OpenapiState.closingReply(state);
+
+            assert replyAck <= replySeq;
+
+            delegate.doHttpReplyAbort(traceId);
+        }
+
+        private void doHttpReplyReset(
+            long traceId)
+        {
+            if (!OpenapiState.replyClosed(state))
+            {
+                doReset(receiver, originId, routedId, replyId, replySeq, replyAck, replyMax,
+                    traceId, authorization, EMPTY_OCTETS);
+
+                state = OpenapiState.closeReply(state);
+            }
+        }
+
+        private void doHttpReplyWindow(
+            long traceId,
+            long authorization,
+            long budgetId,
+            int padding)
+        {
+            replyAck = Math.max(delegate.replyAck - replyPad, 0);
+            replyMax = delegate.replyMax;
+
+            doWindow(receiver, originId, routedId, replyId, replySeq, replyAck, replyMax,
+                traceId, authorization, budgetId, padding + replyPad);
+        }
+    }
+
+    private final class OpenapiStream
```

Comment:
This `OpenapiStream` inner class definition should go before `HttpStream` inner class definition because `OpenapiStream` logically occurs first.